### PR TITLE
Models for Airbus A380, Gulfstream G550, and  MD90

### PR DESCRIPTION
**Pull request by tahaitani**
All models run with SLSQP optimizer

### CODE REVIEW

**Review by ehariton — APPROVED** [review]

File: Gulfstream G550/G550.csv [hunk 1]
```diff
@@ -0,0 +1,162 @@
+aircraft:air_conditioning:mass_scaler,1,unitless,
+aircraft:anti_icing:mass_scaler,1,unitless,
+aircraft:apu:mass_scaler,1.1,unitless,
+aircraft:avionics:mass_scaler,1.2,unitless,
+aircraft:canard:area,0,ft**2,
+aircraft:canard:aspect_ratio,0,unitless,
+aircraft:canard:thickness_to_chord,0,unitless,
+,,,
```
Comment (jkirk5) [inline Gulfstream G550/G550.csv]: clean up extra commas

File: MD90/MD90.csv [hunk 2]
```diff
@@ -0,0 +1,160 @@
+aircraft:air_conditioning:mass_scaler,1,unitless,,
```
Comment (jkirk5) [inline MD90/MD90.csv]: also extra commas in this file

**Review by jkirk5 — CHANGES_REQUESTED** [review]
just some minor formatting requests

File: Gulfstream G550/G550.csv (hunk 1 above)
Comment (tahaitani) [reply Gulfstream G550/G550.csv]: Fixed

File: MD90/MD90.csv (hunk 2 above)
Comment (tahaitani) [reply MD90/MD90.csv]: Fixed and removed all extra commas

**Review by jkirk5 — APPROVED** [review]

**Pull request merged**